src/marketing/product-recommendations.md

```diff
@@ -21,7 +21,31 @@ This dashboard displays a table of previously configured recommendations (if any
 
 See [Recommendation Types]({% link marketing/prex-types.md %}) to learn about the available recommendation types in Magento.
 
-## Backup recommendations {#backup-recommendations}
+## Training the machine learning models
+
+Some recommendations do not use any behavioral data and therefore do not use machine learning models to compute results. You can use those recommendations immediately.
+
+Other recommendations however, require behavioral data from your shoppers to train machine learning models that build personalized recommendations. So when you can start using those types of recommendations on your storefront? It depends. The more data these models have, the more accurate and useful the recommendations will be. Collecting this data takes time. This is referred to as the "Cold Start" problem. The "Cold Start" problem is a measure of how much time a model needs to train before it can be considered high quality. In product recommendations, it translates to waiting for Adobe Sensei to train its machine learning models before deploying Recommendation units on your site. It is therefore in your best interest to deploy data collection to your production site immediately after you [install](https://devdocs.magento.com/recommendations/install-configure.html) the `magento/production-recommendations` module. While data collection is working, you can implement the [remaining tasks necessary](https://devdocs.magento.com/recommendations/implementation.html) to deploy recommendations to your storefront. Ideally, by the time you have finished testing and configuring recommendations, the machine learning models will have collected and computed enough data to build relevant recommendations thus allowing you to deploy the recommendations to your storefront.
```

Comment:
```suggestion
Other recommendation types however, require behavioral data from your shoppers to train machine learning models that build personalized recommendations. So when you can start using those types of recommendations on your storefront? It depends.  This is referred to as the "Cold Start" problem. The "Cold Start" problem is a measure of how much time a model needs to train before it can be considered high quality. In product recommendations, it translates to waiting for Adobe Sensei to train its machine learning models before deploying Recommendation units on your site. The more data these models have, the more accurate and useful the recommendations will be. Collecting this data takes time and will vary based on traffic volume. It is therefore in your best interest to deploy data collection to your production site immediately after you [install](https://devdocs.magento.com/recommendations/install-configure.html) the `magento/production-recommendations` module. While data is collected and ML models are trained, you can implement the [remaining tasks necessary](https://devdocs.magento.com/recommendations/implementation.html) to deploy recommendations to your storefront. Ideally, by the time you have finished testing and configuring recommendations, the machine learning models will have collected and computed enough data to build relevant recommendations thus allowing you to deploy the recommendations to your storefront.
```
We need an ending that inspires more confidence in the user (instead of "ideally"). we should also talk about the case when they DIDN'T wait to start data collection until the end. I would basically say something like wait as long as you can (or willing) but it should be safe to deploy within a few days.